k8s/charts/openebs/templates/deployment-maya-provisioner.yaml

```diff
@@ -23,3 +23,7 @@ spec:
           valueFrom: 
             fieldRef:
               fieldPath: spec.nodeName
+        - name: OPENEBS_MONITOR_URL
+          value: "{{ .Values.provisioner.monitorurl }}"
```

Comment:
These seems to be OK as there can be only URL at a given point in time. Isn't it ?

Reply:
yes. the URL needs to be provided via command line or values.yaml.